### PR TITLE
Update administrator.rst

### PR DESCRIPTION
**Pull request by kichloo**
Fixes #4018

### CODE REVIEW

**Comment by justinegeffen** [comment]
Thank you, @kichloo. One suggestion is naming it something like ``light-install-hindi.rst`` so that it's not confused with the English version which has the same name. Let me know what you think?

**Comment by kichloo** [comment]
@justinegeffen No problem! Done

File: source/guides/administrator.rst [hunk 1]
```diff
@@ -265,3 +265,4 @@ Documentation on early previews and unofficial functionality.
    /install/prod-docker*
    /install/deploy-cloudron*
    /administration/upgrade-script*
+   /administration/light-install.rst
```
Comment (justinegeffen) [inline source/guides/administrator.rst:268]:
```suggestion
   /administration/light-install-hindi.rst
```

**Comment by kichloo** [comment]
@justinegeffen Done

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**